python-api/gstswitch/helpers.py

```diff
@@ -134,10 +137,6 @@ def get_test_video(self):
         """Returns a list of processes acting as video test sources running
```

Comment:
it should actually be removed, because there is already a property `running_tests_video` which does exactly the same.